cmd/lakectl/cmd/branch.go

```diff
@@ -154,7 +154,7 @@ var branchResetCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		clt := getClient()
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```

Comment:
```suggestion
		fmt.Println("Branch: %s", u)
```

Reply:
kept the original format and dropped the call to String()